add note to resolver in nginx reverse proxy docs (#1716)
nextcloud/all-in-one: reverse-proxy.md

    server_name <your-nc-domain>;

    location / {

>         resolver localhost; # Note: if you don't run a dns resolver on localhost:53, you should use another resolver

Reviewer: Ah thar explains things!

Reviewer: ```suggestion
        resolver localhost; # Note: if you don't run a dns resolver on localhost:53, you should use another resolver. Otherwise you need to use 127.0.0.1 in the line below this one instead of localhost
```

Reviewer: Or maybe add the note into the line directly? That if not resolver is defined you need to use 127.0.0.1?

Author: ```suggestion
        resolver localhost; # Note: you need to set a valid dns resolver here or use 127.0.0.1 / [::1] instead of localhost in the line below
```

Reviewer: Yeah looks good, or maybe add the advice in the line below?

Reviewer: ```suggestion
        resolver localhost; # Note: if you don't run a dns resolver on localhost:53, you should use another resolver. See https://stackoverflow.com/a/49642310 for a better explanation
```

>         proxy_pass http://localhost:11000$request_uri;

Author: ```suggestion
        proxy_pass http://localhost:11000$request_uri; # Node: you need to change localhost to 127.0.0.1 / [::1], if you don't use a valid dns resolver in the line above
```

Reviewer: Yeah looks good but should be `Note` instead of `Node`

Reviewer: Also would you need to add two lines if you use both ipv4 and ipv6?

Author: > Also would you need to add two lines if you use both ipv4 and ipv6?

No, but if a system runs ipv6 only it needs [::1] and ipv4 only, it needs 127.0.0.1, if both are enabled it doesn't matter what is used

    server_name <your-nc-domain>;

    location / {
        resolver localhost; # Note: you need to set a valid dns resolver here or use 127.0.0.1 / [::1] instead of localhost in the line below. See https://stackoverflow.com/a/49642310 for a better explanation

>         proxy_pass http://localhost:11000$request_uri; # Note: you need to change localhost to 127.0.0.1 / [::1], if you don't use a valid dns resolver in the line above

Reviewer: ```suggestion
        proxy_pass http://localhost:11000$request_uri; # Note: you need to change localhost to 127.0.0.1 or [::1], if you don't use a valid dns resolver in the line above
```
